com.bitwig.BitwigStudio.yaml

```diff
@@ -0,0 +1,76 @@
+---
+app-id: com.bitwig.BitwigStudio
+command: bitwig-studio
+runtime-version: "20.08"
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+rename-desktop-file: bitwig-studio.desktop
+rename-icon: bitwig-studio
+#rename-appdata-file: 'bitwig-studio.appdata.xml'
+finish-args:
+  # We need MIDI, Audio, USB, DRI, and realtime performances
+  - --device=all
+
+  # Needed to talk with X11, Wayland, pulseaudio and pipewire
+  - --share=ipc
+  - --filesystem=xdg-run/pipewire-0
+  - --socket=pulseaudio
+  - --socket=wayland
```

Comment:
```suggestion
```

Reply:
Why?

Reply:
We don't need it, but I don't understand why it is a problem to have it?